redis_lru/lru.py

```diff
@@ -24,11 +24,13 @@ def __init__(self,
                  max_size=2 ** 20,
                  default_ttl=15 * 60,
                  key_prefix='RedisLRU',
-                 clear_on_exit=False):
+                 clear_on_exit=False,
+                 exceptions=None):
         self.client = client
         self.max_size = max_size
         self.key_prefix = key_prefix
         self.default_ttl = default_ttl
+        self.exceptions = exceptions if type(exceptions) is list else list()
```

Comment:
I think `set` is a better data struct in here, not a list.

Reply:
It has a set now.